packages/cerebral-provider-firebase/src/helpers.js

```diff
@@ -83,3 +83,30 @@ export function createUser (user) {
     photoURL: user.photoURL
   }
 }
+
+export function createReturnPromise (returnPromise, path) {
```

Comment:
createReturnPromise and convertObjectWithTemplates seems generic and useful enough to be exported from 'cerebral/helpers'